flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["i386", "x86_64"]
```

Comment:
i386 has not been supported for years
```suggestion
  "only-arches": [ "x86_64"]
```

Reply:
what about aarch64?

Reply:
I added `aarch64` but will not be able to test it